javascript-timers/interval.js

```diff
@@ -0,0 +1,10 @@
+var $countDown = document.querySelector('h1');
+var timer = 4;
+function countDown() {
+  if (timer <= 4 && timer >= 1) {
+    $countDown.textContent = timer--;
+  } else {
+    $countDown.textContent = '~Earth Beeeelooowww Us~';
+  }
+};
+setInterval(countDown, 1000);
```

Comment:
Since the interval will repeat indefinitely, it's important to stop it when you're done with it.

Reply:
![image](https://user-images.githubusercontent.com/78944582/123720962-24adba00-d83a-11eb-9d0c-184e9067079f.png)